src/_gettsim/functions/loader.py

```diff
@@ -186,10 +218,12 @@ def _create_policy_function_from_decorated_callable(
     """
 
     # Only needed until the directory structure is cleaned up
+    # TODO(@MImmesberger): Remove the removeprefix calls once the directory
```

Comment:
I actually think we should do that right now. We will not merge this one into main, so we want to have the full machinery in place before tackling the renaming. Or am I missing something? Same for the other occurrence.

Reply:
And that was the right thought at the time we wanted to merge into main!

Reply:
We haven't settled on a specific directory structure yet. I propose:

```text
|-- src
|     |-- gettsim
|     |       |-- ...
|     |-- _gettsim
|     |       |-- infrastructure (alternatively: ttsim?)
|     |       |        |-- aggregation_jax.py
|     |       |        |-- aggregation_numpy.py
|     |       |        |-- aggregation.py
|     |       |        |-- gettsim_typing.py
|     |       |        |-- piecewise_functiony.py
|     |       |        |-- policy_environment.py
|     |       |        |-- policy_environment_postprocessor.py
|     |       |        |-- interface.py
|     |       |        |-- shared.py
|     |       |        |-- time_conversion.py
|     |       |        |-- vectorization.py
|     |       |-- parameters
|     |       |-- taxes_and_transfers
|     |       |        |-- demographics
|     |       |        |-- einkommensteuer
|     |       |        |-- arbeitslosengeld_2
|     |       |        |-- wohngeld
|     |       |        |-- ...
|     |       |        |-- groupings.py
|     |       |-- config.py
|     |       |-- synthetic.py
|     |       |-- visualization.py
|     |-- _gettsim_tests
|     |       |-- ...
```

Reply:
Great. Again, I think I would be even more radical, now that we are at it:

```text
|-- src
|     |-- gettsim
|     |       |-- ...
|     |-- _gettsim
|     |       |-- ttsim
|     |       |        |-- config.py
|     |       |        |-- aggregation_jax.py
|     |       |        |-- aggregation_numpy.py
|     |       |        |-- aggregation.py
|     |       |        |-- typing.py
|     |       |        |-- piecewise_functiony.py
|     |       |        |-- policy_environment.py
|     |       |        |-- policy_environment_postprocessor.py
|     |       |        |-- interface.py
|     |       |        |-- shared.py
|     |       |        |-- time_conversion.py
|     |       |        |-- vectorization.py
|     |       |        |-- visualize_dag.py
|     |       |-- de
|     |       |        |-- config.py
|     |       |        |-- parameters
|     |       |        |-- demographics
|     |       |        |-- einkommensteuer
|     |       |        |-- arbeitslosengeld_2
|     |       |        |-- wohngeld
|     |       |        |-- ...
|     |       |        |-- groupings.py
|     |       |        |-- synthetic.py
|     |-- _gettsim_tests
|     |       |-- ttsim
|     |       |        |-- ...
|     |       |-- de
|     |       |        |-- ...
```

That is,
- ttsim suggestion is cool, change taxes_and_transfers accordingly
- config and synthetic need to be split up among these two
- just typing, not gettsim_typing
- visualize_dag (we should probably rename the function, too?)
- make parameters part of de

Probably easiest to inject this as [step 3 in the grand PR](https://github.com/iza-institute-of-labor-economics/gettsim/pull/804#issue-2735500579) given the work you put in so far?